pints/tests/test_opt_optimisation_controller.py

```diff
@@ -76,56 +73,55 @@ def test_logging(self):
         opt.set_max_unchanged_iterations(None)
         opt.set_log_interval(3)
         opt.set_max_iterations(10)
-        self.assertEqual(opt.max_iterations(), 10)
         with StreamCapture() as c:
             opt.run()
-            log_should_be = (
-                'Maximising LogPDF\n'
-                'Using Exponential Natural Evolution Strategy (xNES)\n'
-                'Running in sequential mode.\n'
-                'Population size: 6\n'
-                'Iter. Eval. Best      Time m:s\n'
-                '0     6     -4.140462   0:00.0\n'
-                '1     12    -4.140462   0:00.0\n'
-                '2     18    -4.140462   0:00.0\n'
-                '3     24    -4.140462   0:00.0\n'
-                '6     42    -4.140462   0:00.0\n'
-                '9     60    -4.140462   0:00.0\n'
-                '10    60    -4.140462   0:00.0\n'
-                'Halting: Maximum number of iterations (10) reached.\n'
-            )
-            self.assertEqual(log_should_be, c.text())
+        lines = c.text().splitlines()
+        self.assertEqual(lines[0], 'Maximising LogPDF')
+        self.assertEqual(
+            lines[1], 'Using Exponential Natural Evolution Strategy (xNES)')
+        self.assertEqual(lines[2], 'Running in sequential mode.')
+        self.assertEqual(lines[3], 'Population size: 6')
+        self.assertEqual(lines[4], 'Iter. Eval. Best      Time m:s')
+        self.assertEqual(lines[5][:-3], '0     6     -4.140462   0:0')
+        self.assertEqual(lines[6][:-3], '1     12    -4.140462   0:0')
+        self.assertEqual(lines[7][:-3], '2     18    -4.140462   0:0')
+        self.assertEqual(lines[8][:-3], '3     24    -4.140462   0:0')
+        self.assertEqual(lines[9][:-3], '6     42    -4.140462   0:0')
+        self.assertEqual(lines[10][:-3], '9     60    -4.140462   0:0')
+        self.assertEqual(lines[11][:-3], '10    60    -4.140462   0:0')
+        self.assertEqual(
+            lines[12], 'Halting: Maximum number of iterations (10) reached.')
 
         # Invalid log interval
         self.assertRaises(ValueError, opt.set_log_interval, 0)
 
         # Test with error measure
         r = pints.toy.RosenbrockError()
         x = np.array([1.01, 1.01])
-        opt = pints.OptimisationController(r, x, method=method)
+        opt = pints.OptimisationController(r, x, method=pints.SNES)
```

Comment:
Just curious why changing this to SNES?

Reply:
Just to make the test output a bit different from the previous test. In case we have an evil programmer who just hardcodes the method name :D